### ISSUE TITLE
The place to find the Client ID and Secret is described in this link (which is broken!)

### ISSUE THREAD

**Issue by shadow7412** [rospogrigio/localtuya#1803]
It looks like the authentication flow in home assistant has changed to using a "user code" from the tuya app.

Can we please have the documentation updated to show the required steps, instead of linking to the home assistant docs?

**Comment by meluvalli** [comment]
I also had this problem, but figured it out.  From the Tuya developer account webpage, click "Cloud".
Then On the right side of the page, click "Open Project".
This will bring you to the Overview page of your project.  Your Client/ID and Secret are there.

**Comment by kaphadley** [comment]
The setup documentation wrongly assumes that the external link's content will remain the same and that the user is familiar with every field. The setup form should provide clear instruction for each field, when it's not done this information should at least be available in the documentation.

**Comment by shadow7412** [comment]
Exactly. This isn't really a request for help, but reporting a problem in the documentatation.

**Comment by kaovilai** [comment]
Cloud > Project Management > Create Project :D
